composer.json

```diff
@@ -106,7 +106,7 @@
   "require-dev": {
     "cache/integration-tests": "dev-master",
     "codeception/codeception": "^4.1.12",
-    "codeception/module-symfony": "^1.5.0",
+    "codeception/module-symfony": "^1.6.0",
```

Comment:
since we bumped it to 1.6.x, let's remove line `"codeception/module-symfony": "1.5.1"` from conflicts 😉

Reply:
Good catch, thanks a lot!